### PR TITLE
US-22: Adding Dev focus related backend changes

### PR DESCRIPTION
**Pull request by achaur11**

### CODE REVIEW

File: taigaProject/src/taigaApi/task/get_task_history.py [hunk 1]
```diff
@@ -145,4 +157,124 @@ def get_task_cycle_time(project_id, auth_token):
...
+    for member in members:
+        tasks = task_by_member_in_date_range(project_id, member, from_date, to_date, auth_token)
+        all_tasks = all_tasks + tasks
+
+    return all_tasks
+
+
+def member_tasks(project_id, from_date, to_date, members, auth_token):
+    member_tasks = {}
+    # for member in members:
```
Comment (shahih25) [inline taigaProject/src/taigaApi/task/get_task_history.py]: Remove commented lines

Comment (achaur11) [reply taigaProject/src/taigaApi/task/get_task_history.py]: Removed.

**Review by shahih25 — APPROVED** [review]
lgtm

**Comment by spark290** [comment]
LGTM

**Pull request merged**